adam-core/src/main/scala/org/bdgenomics/adam/util/ReferenceContigMap.scala

```diff
@@ -20,10 +20,18 @@ package org.bdgenomics.adam.util
 import org.apache.spark.rdd.RDD
 // NOTE(ryan): this is necessary for Spark <= 1.2.1.
 import org.apache.spark.SparkContext._
-import org.bdgenomics.adam.models.ReferenceRegion
+import org.bdgenomics.adam.models.{ ReferenceRegion, SequenceDictionary, SequenceRecord }
 import org.bdgenomics.formats.avro.NucleotideContigFragment
 
 case class ReferenceContigMap(contigMap: Map[String, Seq[NucleotideContigFragment]]) extends ReferenceFile {
+
+  /**
+   * Sequence dictionary extracted from contigMap
+   * @see ReferenceFile.scala
```

Comment:
this @see won't work in javadoc/scaladoc, it should be `@see ReferenceFile`, although generally I don't find doc links all that useful.